project-base/tests/ShopBundle/Functional/Model/Product/Availability/ProductAvailabilityRecalculatorTest.php

```diff
@@ -30,7 +30,7 @@ public function testRecalculateOnProductEditNotUsingStock()
         $productData->availability = $this->getReference(AvailabilityDataFixture::AVAILABILITY_ON_REQUEST);
 
         $productFacade->edit($productId, $productData);
-        $productAvailabilityRecalculator->runAllScheduledRecalculations();
```

Comment:
why this change? (and the following similar ones) Have I missed something? The whole user story is about visibility recalculations but you have removed the marking for availability recalculation in `Product::edit()` method...if this is intentional, please explain it in the commit message

Reply:
This change has been removed